### DISCUSSION TITLE
How to set the proportion of old and new data sets

### DISCUSSION THREAD

**Discussion by foreverfreedown** [deepmodeling/deepmd-kit#627] [Q&A]
![image](https://user-images.githubusercontent.com/49577178/118141876-6b875200-b43c-11eb-834e-daf9eb634070.png)
I saw on the public account of depth potential energy how to set the ratio between the old and new data sets, but I did not understand his instructions below how to achieve the ratio of the old and new data sets to 9:1, I hope you can help me.
Thanks!

**Comment by tuoping** [comment]
![捕获](https://user-images.githubusercontent.com/80671886/118207517-6e636080-b497-11eb-8df3-0ba7d42272dc.PNG)
Hope this would help. It's captured from lib/python3.8/site-packages/deepmd/DataSystem.py
